### ISSUE TITLE
Wrong git date format - why?

### ISSUE THREAD

**Issue by l3nz** [l3nz/say_cheez_ex#11]
Sometimes I see

```
Compiling 121 files (.ex)
fatal: unknown date format format:%Y%m%d.%H%M
fatal: unknown date format format:%Y%m%d.%H%M
```

See https://stackoverflow.com/questions/7853332/how-to-change-git-log-date-formats

**Comment by l3nz** [comment]
On CentOS7, we have git 1.8.3.1, where:

```
git log --oneline -n 1 --pretty=%cd --date=format:%Y%m%d.%H%M
fatal: unknown date format format:%Y%m%d.%H%M
```

**Comment by l3nz** [comment]
This works:

```
git log --oneline -n 1 --pretty=%cd --date=iso
2023-02-15 08:50:19 +0100
```

And we could make it into variables.

**Comment by l3nz** [comment]
It was actually fixed as for #7
